elementor/elementor
# Editor/Screenshots: New - capture screenshot on publish

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Description
An explanation of what is done in this PR

* On publish it creates a screenshot of the current view of the post

## Test instructions
This PR can be tested by following these steps:

* Open a post with the editor
* Make some changes as you wish
* Publish the post
* Make sure that screenshot is created at the upload directory
* Make sure that the screenshot look the same as the preview of the post.

## Quality assurance


Fixes #
